components/gc-features/_base.scss

```diff
@@ -22,6 +22,7 @@
 
 	.well {
 		border-radius: 0;
+		height: 100%;
```

Comment:
This need to be tested: Text-zoom. But we should not need to use any "height:100%".

More rational will be needed to understand if this don't reveal a bug with the equal height CSS feature we have.

Reply:
Removed `height: 100%` and reinstated `.eqht-trgt`